database/postgreSQL.js

```diff
@@ -0,0 +1,10 @@
+const { Pool } = require('pg');

```

Comment:
I haven't read a lot into postgres, but wouldn't you have to connect to a client before creating a pool?